scout/server/blueprints/alignviewers/views.py

```diff
@@ -145,6 +152,24 @@ def igv():
         "displayMode": "EXPANDED",
     }
 
+    display_obj["clinvar_snvs"] = {
+        "name": "Clinvar SNVs",
```

Comment:
Could we make that ClinVar? 🐫 🐫 🙈

Reply:
> Could we make that ClinVar? 🐫 🐫 🙈

What's that, camels?, I don't understand!

Reply:
Sorry; as in CamelCase!

Reply:
Btw, the svs don't render well on stage for me? Anything one can do with expandedRowHeight and maxRows??

Reply:
![Screenshot 2020-02-25 at 14 25 49](https://user-images.githubusercontent.com/758570/75251372-c3f01680-57da-11ea-9aab-285502cc05eb.png)

Reply:
I'll do it. Here is a case with SVs, to the the new track: https://scout-stage.scilifelab.se/cust000/643594-300M/

Reply:
What's the camelcase?

Reply:
![Screenshot 2020-02-25 at 14 28 21](https://user-images.githubusercontent.com/758570/75251571-20533600-57db-11ea-9370-e3f9a88e380c.png)

Reply:
Good to know! 👍

Reply:
![Screenshot 2020-02-25 at 14 29 00](https://user-images.githubusercontent.com/758570/75251690-6dcfa300-57db-11ea-8f96-353d3a83f8f8.png)